tapeforms/templates/tapeforms/fields/foundation_fieldset.html

```diff
@@ -0,0 +1,25 @@
+<fieldset class="{{ container_css_class }} {{ container_css_class }}-{{ field_name }} form-widget-{{ widget_class_name }}{% if required %} is-required{% endif %}{% if errors %} has-errors{% endif %}{% if field.css_classes %} {{ field.css_classes }}{% endif %}">
```

Comment:
Basically, this template differs from the fields/foundation.html in two points:
* DOM Element used a the container (div vs. fieldset)
* DOM Element used for the text (label vs. legend)

For the label I'm fine with the change but I'm thinking about having a template variable to replace the "container html element" and then extending the field context in the foundation mixin.

Something like `<{{ container_element|default:'div' }} ......`. + In the mixin, we could add a context `'container_element': 'fieldset'` (in `get_field_context`)

This would help us remove the extra template or at least reducing the compexity of the template (just overide the label block vs. the whole template).

I'm keen on your optinion on that! :-)

Reply:
I agree too, it could be great to not have this extra template. I will try to purpose something as you suggest, but maybe in another PR?